model/src/main/kotlin/config/PathExclude.kt

```diff
@@ -19,6 +19,8 @@
 
 package org.ossreviewtoolkit.model.config
 
+import com.fasterxml.jackson.annotation.JsonInclude
```

Comment:
Commit message title: "PathExclude~~s~~"
And please also mention that this commit also changes the serialization to not serialize empty messages anymore.

Reply:
Would be nice if we could cover this with a test, but it seems the only place were `PathExclude` (de-)serialization is currently tested is `RepositoryConfigurationTest`.